docs/core/development/policy/release-process.rst

```diff
@@ -69,28 +86,38 @@ To release Twisted, we
 Prepare for a release
 ---------------------
 
-#. Check for any ​regressions using `Trac regression report <https://twistedmatrix.com/trac/report/26>`_
+#. Check for any regressions using `Trac regression report <https://twistedmatrix.com/trac/report/26>`_
 
-#. Any regression should be fixed and merged trunk before making the release branch
+#. Any regression should be fixed and merged into trunk before making the release branch
 
 #. Choose a version number.
+   $RELEASE will be something like 21.7.0 for a major release.
+   $RELEASE will be something like 21.7.1 for a bugfix release.
 
 #. File a ticket in Trac called "Release $RELEASE" and assign it to yourself.
 
-#. Make a branch for the release and include the ticket number in the name (4290 is Trac ticket ID):
+#. Make a branch for the release.
+   It's very important to use `release-$RELEASE-$TRAC_ID` as the branch name (4290 is Trac ticket ID, 21.7.0 is the release number) as this is used as a hint for CI:
 
    - ``git fetch origin``
    - ``git checkout origin/trunk``
-   - ``git checkout -b release-$RELEASE-4290``
+   - ``git checkout -b release-21.7.0-4290``
 
 
 How to do a release candidate
 -----------------------------
 
 
+This section described the required to create the first release candidate.
```

Comment:
I don't understand this line? A word missing?

Reply:
```suggestion
This section describes the steps and requirements for creating the first release candidate.
```

Reply:
Nice

Reply:
Yah that's it, "required to create" tripped me up.